examples/run_full_prep.py

```diff
@@ -168,7 +172,7 @@
 
 EEG_new_matlab = sio.loadmat(fname_mat2)
 EEG_new_matlab = EEG_new_matlab["save_data"]
-EEG_new = prep.EEG_new
+EEG_new = removeTrend(prep.EEG_raw, sample_rate=prep.sfreq) * 1e6
```

Comment:
@a-hurst could it be that you didn't rebase this PR or did a rebase that went wrong? Because we just remove this in #102

Reply:
Nope, I somehow missed the full prep example during #102 so I'm fixing it here. My bad!

Reply:
To clarify: beforehand, these `PrepPipeline` EEG data attributes were all in microVolts which matches EEGLAB convention, so the `* 1e6` wasn't needed to compare with MATLAB results. Now that those attributes are in MNE-standard Volts, we need to convert them to uV to compare with EEGLAB data the way we do here.

Reply:
Thanks!